### ISSUE TITLE
Discussion - Improvement to Documentation Examples - Difficultly Understanding Imports

### ISSUE THREAD

**Issue by DamonGuzman** [dateutil/dateutil#1365]
Thank you so much for all of the amazing work on this library! 

For some context: 
I opened the documentation for the first time about 20 minutes ago and I was immediately having difficulty understanding how to use this library. Because the examples are using `from x import *` syntax, it requires a lot of mental effort to identify what is coming from the library vs what is coming from Python's `datetime` since they are frequently imported together. I personally am not an expect in Python's `datetime` module and while looking for simple solutions I saw recommendations for using this library alongside the native `datetime` module. Perhaps, I'm not the intended audience of this package and that's why the examples are not immediately intuitive to me. 

A direct example from [the docs](https://dateutil.readthedocs.io/en/stable/index.html#quick-example)

```
from dateutil.relativedelta import *
from dateutil.easter import *
from dateutil.rrule import *
from dateutil.parser import *
from datetime import *
now = parse("Sat Oct 11 17:13:46 UTC 2003")
today = now.date()
year = rrule(YEARLY,dtstart=now,bymonth=8,bymonthday=13,byweekday=FR)[0].year
rdelta = relativedelta(easter(year), today)
```

I don't immediately identify where the following things are coming from. I had to double check the datetime docs to make sure they didn't have any imports with the same name.

```
from dateutil.relativedelta import *
from dateutil.easter import *
from dateutil.rrule import *
from dateutil.parser import *
from datetime import *
             vvv 
now = parse("Sat Oct 11 17:13:46 UTC 2003")
today = now.date()
             vv                                                    vv
year = rrule(YEARLY,dtstart=now,bymonth=8,bymonthday=13,byweekday=FR)[0].year
                  vvv  
rdelta = relativedelta(easter(year), today)
```

I think if the examples avoided `import *` as a general rule, it would make things clearer for myself and anyone in my situation. Then again, I may not be the intended audience for this library. Thanks for your time!

**Comment by pganssle** [comment]
Yes, I tend to agree with this. I may have "drunk the Kool aid" a bit since working at Google, but I have come to strongly prefer the [Google Python style guide's recommendation](https://google.github.io/styleguide/pyguide.html#22-imports) about imports, where you import modules and use them in place.

Might be a bit unwieldy and verbose for `relativedelta.relativedelta`, but 🤷

**Comment by arel** [comment]
I was about to suggest the same pull request. :) And, thanks for the pointer to Google Python style guide recommendation.
